polars/lazy/expr/index.ts

```diff
@@ -887,6 +887,17 @@ export interface Expr
     default_?: Expr | string | number | (number | string)[];
     returnDtype?: DataType;
   }): Expr;
+  replace(
+    old: Expr | string | number | (number | string)[],
+    new_: Expr | string | number | (number | string)[],
```

Comment:
```suggestion
    oldValues: Expr | string | number | (number | string)[],
    newValues: Expr | string | number | (number | string)[],
```

Reply:
This change would involve changing existing options. 
We would need to deprecate `new_` and 'old' options and they would be different from Python Polars.

Reply:
that should be fine. It's fine if we deviate from python where it makes sense. `new` is a reserved word in JS, so i'd like to avoid it, and think `newValues` is better than `new_`.